cln-rpc/src/model.rs

```diff
@@ -2494,6 +2512,7 @@ pub mod responses {
 
 	#[derive(Clone, Debug, Deserialize, Serialize)]
 	pub struct FeeratesOnchain_fee_estimates {
+		// ^^ Similar naming/casing pattern
```

Comment:
I thought that casing in msggen was weird too, and may be local to my issues, but it looks like several places in the model use the same casing.  Like here.

Reply:
Yes it is an unrelated bug from this PR, maybe we should track with an issue?